assembl/static2/js/app/components/login/signupForm.jsx

```diff
@@ -5,24 +5,27 @@ import { connect } from 'react-redux';
 import { Translate, I18n } from 'react-redux-i18n';
 import { form, FormGroup, FormControl, Button, Checkbox } from 'react-bootstrap';
 import { Link } from 'react-router';
+import * as lodashGet from 'lodash/get';
```

Comment:
Please just use `import get from 'lodash/get';` like we do in all the current code base.

Reply:
Indeed, sorry for the noise. :)